env

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
I guess that `BASH_SOURCE[0]` could be a relative path, so you need to use `cd` and `pwd` to get the absolute path?

Reply:
Yea I think that's it. Maybe I should leave a note to the stackoverflow place I copied it from.